rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_raw.cpp

```diff
@@ -4,6 +4,35 @@
 
 LOG_CHANNEL(sys_net);
 
+struct socket_raw_logging
+{
+	socket_raw_logging() = default;
+
+	socket_raw_logging(const socket_raw_logging&) = delete;
+	socket_raw_logging& operator=(const socket_raw_logging&) = delete;
+
+	bool getpeername = false;
+	bool listen = false;
+	bool bind = false;
+	bool getsockname = false;
+	bool getsockopt = false;
+	bool setsockopt = false;
+	bool recvfrom = false;
+	bool sendto = false;
+	bool sendmsg = false;
+	bool close = false;
+	bool shutdown = false;
+	bool poll = false;
+	bool select = false;
```

Comment:
Instead of a member for each, I had put this struct in a template.
Each access, put an incomplete typename describing the operation:

g_fxo->get<log_once_t<class of_send_message>>().logged.exchange(true)

For example. Do note that of_send_message does not need to be declared anywhere.